src/components/Location/LocationTree.tsx

```diff
@@ -105,6 +105,8 @@ export function LocationTree({
   isLatest,
   showTimeline = false,
 }: LocationPathProps) {
+  const completed = isLatest && endTime;
+
```

Comment:
do we really need the `isLatest` check to know whether it is completed or not? 🤔 

we are planning to introduce multiple active locations for an encounter. i.e., for example, an encounter would have an active location and multiple planned locations in the future as well.

```suggestion
  const isCompleted = !!endTime;

```